src/color.cpp

```diff
@@ -210,157 +210,157 @@ void color_manager::load_default()
     //        Color         Name      Color Pair      Invert
     add_color( def_c_black, "c_black", color_pair( 30 ), def_i_black );
     add_color( def_c_white, "c_white", color_pair( 1 ).bold(), def_i_white );
-    add_color( def_c_ltgray, "c_ltgray", color_pair( 1 ), def_i_ltgray );
-    add_color( def_c_dkgray, "c_dkgray", color_pair( 30 ).bold(), def_i_dkgray );
+    add_color( def_c_ltgray, "c_light_gray", color_pair( 1 ), def_i_ltgray );
```

Comment:
Probably should change the enum def_c_* as well to be consistend.

Reply:
Will address this in subsequent PR #22572.